src/app/shared/header/header.component.html

```diff
@@ -156,3 +155,13 @@
     </a>
   </button>
 </mat-menu>
+
+<!-- Search Menu Dropdown -->
+<mat-menu #search="matMenu">
+  <button mat-menu-item>
+    <a class="nav-link" routerLink="/graphql" routerLinkActive="active">Search</a>
+  </button>
+  <button mat-menu-item>
+    <a class="nav-link" routerLink="/globalsearch" routerLinkActive="active">Global Search</a>
```

Comment:
can we rename this to just Search

Reply:
we have just 'Search' for the graphql there